src/Fuzz.elm

```diff
@@ -343,6 +343,14 @@ list fuzzer =
 
 listShrinkHelp : List (RoseTree a) -> RoseTree (List a)
 listShrinkHelp listOfTrees =
+    {- This extends listShrinkRecurse algorithm with an attempt to shrink directly to the empty list. -}
+    case listShrinkRecurse listOfTrees of
+        Rose root children ->
+            Rose root (Lazy.List.cons (RoseTree.singleton []) children)
```

Comment:
I like to avoid `case` expressions with only one case. I *think* this should work instead: `let (Rose root children) = listShrinkRecurse listOfTrees in ...`